Misc/NEWS.d/next/Library/2021-05-12-13-18-28.bpo-44115.Y11_u8.rst

```diff
@@ -0,0 +1 @@
+improve conversions to fractions
```

Comment:
This is very vague: what kind of conversions are allowed now that weren't before?

Also, this change should have new unit tests.

Reply:
I just added some unit tests